test/integration/test_personality_insights_v3.rb

```diff
@@ -9,10 +9,13 @@ class PersonalityInsightsV3Test < Minitest::Test
     include Minitest::Hooks
     attr_accessor :service
     def before_all
+      authenticator = IBMCloudSdkCore::IamAuthenticator.new(
+        apikey: ENV["PERSONALITY_INSIGHTS_APIKEY"]
+      )
       @service = IBMWatson::PersonalityInsightsV3.new(
         version: "2017-10-13",
-        iam_apikey: ENV["PERSONALITY_INSIGHTS_APIKEY"],
-        url: ENV["PERSONALITY_INSIGHTS_URL"]
+        url: ENV["PERSONALITY_INSIGHTS_URL"],
+        authenticator: authenticator
```

Comment:
do we need to pass in url here and in the other tests?

Reply:
we don't need to but it doesn't hurt to pass it. So I am fine either ways.